WebSTR/WebSTR.py

```diff
@@ -147,7 +147,7 @@ def main():
         server.config.update(
             TEMPLATES_AUTO_RELOAD=True
         )
-    server.run(debug=FLASK_DEBUG, host=args.host, port=args.port)
```

Comment:
same as before, we should keep this as is and just set the FLASK_DEBUG environment variable for debugging